mvpa2/clfs/distance.py

```diff
@@ -39,8 +39,8 @@ def absmin_distance(a, b):
     return max(abs(a-b))
 
 
-def manhatten_distance(a, b):
-    """Return Manhatten distance between a and b
+def manhattan_distance(a, b):
```

Comment:
ho ho -- nice catch ;)

but since it is a part of the API, we need to do such change 'gracefully'
```python

@deprecated("use correctly spelled manhattan_distance instead")
def manhatten_distance(a, b):
    return manhattan_distance(a, b):
```